openwpm/storage/parquet_schema.py

```diff
@@ -10,6 +10,25 @@
 
 PQ_SCHEMAS = dict()
 
+fields = [
+    pa.field("task_id", pa.int64(), nullable=False),
+    pa.field("manager_params", pa.string(), nullable=False),
+    pa.field("openwpm_version", pa.string(), nullable=False),
+    pa.field("browser_version", pa.string(), nullable=False),
+    pa.field("instance_id", pa.uint32(), nullable=False),
+    # Omitting start_time as I couldn't figure out how to implement it
```

Comment:
We should remove this comment and file a bug if we think it's important.

I don't think it is important, and I'm happy to just drop it from `schema.sql` as well to keep the schemas consistent. The start time is self-evident by the timestamps on any of the collected data.

Reply:
I think it's convinient to have this information. E.g. if you want to measure how websites change over time, being able to group by day the crawl ran, seems like something you'd want to have accessible.

Reply:
https://github.com/mozilla/OpenWPM/issues/862